packages/mobile-top-nav/src/css/wayback-search.js

```diff
@@ -0,0 +1,75 @@
+import { css } from 'lit-element';
+
+export default () => {
```

Comment:
That's a fun lil context switch when looking at this file's directory path. css -> js